src/driver.rs

```diff
@@ -271,7 +271,9 @@ pub fn main() {
                 },
                 _ => Some(s.to_string()),
             })
+            // FIXME: remove this line in 1.79 to only keep `--cfg clippy`.
             .chain(vec!["--cfg".into(), r#"feature="cargo-clippy""#.into()])
```

Comment:
@Urgau Can I ask you to remove this line as part of the stabilization PR of the `-Zcheck-cfg` feature in `rust-lang/rust` please? That way, the time people have to transition is as long as possible.

Reply:
Sure, no problem.